CppCoreGuidelines.md

```diff
@@ -2329,7 +2329,7 @@ In fact, C++98's standard library already used this convenient feature, because
 For example, given a `set<string> myset`, consider:
 
     // C++98
-	result = myset.insert( “Hello” );
```

Comment:
Note: removed some Unicode whitespace characters not visible in the diff but showing as squares when the doc is rendered.